charts/k3s/values.yaml

```diff
@@ -159,7 +159,7 @@ syncer:
 # Virtual Cluster (k3s) configuration
 vcluster:
   # Image to use for the virtual cluster
-  image: rancher/k3s:v1.26.0-k3s1
+  image: rancher/k3s:v1.27.3-k3s1
```

Comment:
@matskiv I see that you updated this to fix the generic hooks - https://github.com/loft-sh/vcluster-pro/blame/main/charts/k3s/values.yaml#L166

If we and either merge this, or if you're not sure about it we can make this "Pro only" in the pro code

Reply:
I think that was committed by mistake, but it should be fine to just merge it.